### PR TITLE
feat(multipledatabase): add audit events for multiple databases

### PR DESCRIPTION
**Pull request by zijiacj**
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by MarkPotato777 — DISMISSED** [review]
LGTM

**Review by smallsheeeep — CHANGES_REQUESTED** [review]
Some comments request changes.
BTW: 
1. Should we add new APIs defined in `DatabaseChangeController` into audit?
2. We should add new notification for multiple database change in `notification-policy-metadata.yaml`.

**Review by JessieWuJiexi — DISMISSED** [review]

File: server/odc-core/src/main/resources/i18n/BusinessMessages.properties [hunk 1]
```diff
@@ -207,6 +208,7 @@ com.oceanbase.odc.AuditEventAction.CREATE_ONLINE_SCHEMA_CHANGE_TASK=Create onlin
 com.oceanbase.odc.AuditEventAction.CREATE_APPLY_PROJECT_PERMISSION_TASK=Create apply project permission task
 com.oceanbase.odc.AuditEventAction.CREATE_APPLY_DATABASE_PERMISSION_TASK=Create apply database permission task
 com.oceanbase.odc.AuditEventAction.STOP_ASYNC_TASK=Stop database Change task
+com.oceanbase.odc.AuditEventAction.STOP_MULTIPLE_ASYNC_TASK=Stop multiple database Change task
```
[the remark this answers is not in the document]
Comment (zijiacj) [reply server/odc-core/src/main/resources/i18n/BusinessMessages.properties:211]: image
The last submission has been added it

File: server/odc-core/src/main/resources/i18n/BusinessMessages_zh_TW.properties [hunk 2]
```diff
@@ -208,6 +209,7 @@ com.oceanbase.odc.AuditEventAction.CREATE_ONLINE_SCHEMA_CHANGE_TASK=創建無鎖
 com.oceanbase.odc.AuditEventAction.CREATE_APPLY_PROJECT_PERMISSION_TASK=創建申請項目權限任務
 com.oceanbase.odc.AuditEventAction.CREATE_APPLY_DATABASE_PERMISSION_TASK=創建申請數據庫權限任務
 com.oceanbase.odc.AuditEventAction.STOP_ASYNC_TASK=停止數據庫變更任務
+com.oceanbase.odc.AuditEventAction.STOP_MULTIPLE_ASYNC_TASK=停止多庫變更任務
```
[the remark this answers is not in the document]
Comment (zijiacj) [reply server/odc-core/src/main/resources/i18n/BusinessMessages_zh_TW.properties:212]: yes,I will do it

File: server/starters/desktop-starter/src/main/java/com/oceanbase/odc/service/audit/DesktopAuditEventHandler.java [hunk 3]
```diff
@@ -35,6 +35,7 @@ public class DesktopAuditEventHandler implements AuditEventHandler {
 
     private final List<AuditEventAction> supportedAuditEventActionInClientMode =
             Arrays.asList(AuditEventAction.STOP_ASYNC_TASK,
+                    AuditEventAction.STOP_MULTIPLE_ASYNC_TASK,
```
[the remark this answers is not in the document]
Comment (zijiacj) [reply server/starters/desktop-starter/src/main/java/com/oceanbase/odc/service/audit/DesktopAuditEventHandler.java]: have removed it

**Review by smallsheeeep — DISMISSED** [review]

File: server/odc-core/src/main/resources/i18n/BusinessMessages.properties [hunk 4]
```diff
@@ -194,6 +194,7 @@ com.oceanbase.odc.AuditEventAction.EXECUTE_TASK=Execute task
 com.oceanbase.odc.AuditEventAction.APPROVE=Approve
 com.oceanbase.odc.AuditEventAction.REJECT=Reject
 com.oceanbase.odc.AuditEventAction.CREATE_ASYNC_TASK=Create database Change task
+com.oceanbase.odc.AuditEventAction.CREATE_MULTIPLE_ASYNC_TASK=Create Multiple database Change task
```
Comment (JessieWuJiexi) [inline server/odc-core/src/main/resources/i18n/BusinessMessages.properties:197]: Multiple -> multiple

**Review by Jane201510 — APPROVED** [review]

**Comment by zijiacj** [comment]
> Some comments request changes. BTW:
> 
> 1. Should we add new APIs defined in `DatabaseChangeController` into audit?
> 2. We should add new notification for multiple database change in `notification-policy-metadata.yaml`.

1. have  define template create 、delete and update into audit
2. have done

**Comment by zijiacj** [comment]
> Some comments request changes. BTW:
> 
> 1. Should we add new APIs defined in `DatabaseChangeController` into audit?
> 2. We should add new notification for multiple database change in `notification-policy-metadata.yaml`.

1. have  define template create 、delete and update into audit
2. have done

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by smallsheeeep — APPROVED** [review]
LGTM

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**